src/main/resources/alma/common/fields.xml

```diff
@@ -675,7 +680,7 @@
           <constant value="Aufgegangen in" />
         </data>
         <data source="78505.t">
-          <constant value="Teilweise aufgegangen in" />
+        <any flushWith="700[013] ">
         </data>
```

Comment:
Not sure what this field has to do with this issue? Also formatting.

Reply:
you are right, this can be deleted.